topic0/src/topic0_KeyPoint3/HospitalFactory.java

```diff
@@ -0,0 +1,13 @@
+package topic0_KeyPoint3;
+
+public class HospitalFactory {
+	
+	public Hospital getHospital(String type) {
+		if ("about children".equals(type)) {
```

Comment:
Remember we can use equalsIgnoreCase() and don't rely in user or clients lower or upper cases.

Also "about children" could be a constant.